components.js

```diff
@@ -328,6 +328,10 @@ var components = {
 			"require": "clike",
 			"owner": "chrischares"
 		},
+		"system_verilog": {
```

Comment:
Would you mind naming the language with a dash instead of an underscore?
Or maybe just name it `verilog`, since this is the base language?